.github/workflows/brakeman.yml

```diff
@@ -1,3 +1,18 @@
+# USAGE
+# -----
+#
+# on:
+#   workflow_dispatch: {}
+#   push:
+#     branches: [main]
+#
+# jobs:
+#   security-analysis:
+#     name: Security Analysis
+#     uses: alphagov/govuk-infrastructure/.github/workflows/brakeman.yml@main
+
+# REUSABLE WORKFLOW
+# -----------------
```

Comment:
I'm leaning toward removing these usage docs from all the workflows - I don't think they're helpful as you wouldn't use them verbatim, another thing to keep up to date, and it's not difficult to discern the usage from actual workflow definition.

Reply:
Done :)